app/controllers/mixins/actions/provider_actions/mass_transform.rb

```diff
@@ -14,7 +14,7 @@ def vm_transform_mass
               :target_kls => Provider.name
```

Comment:
Think this is a bug :laughing: should be `provider.name` since we're passing in the `:id` of an `ExtManagementSystem` not a `Provider`